app/solution_service.py

```diff
@@ -0,0 +1,13 @@
+from sympy.parsing.latex import parse_latex
+from sympy import symbols, solve
+
+def is_correct(question, attempt):
+    ## This method will certainly grow in complexity to handle other
+    ## problem types
+
+    x = symbols("x")
```

Comment:
dangerous if there is an `x` in the question?

Reply:
yeah - I'll make changes to make more agnostic